### PR TITLE
Create Me Fragment UI

### PR DESCRIPTION
**Pull request by duccnv-1684**
https://edu-redmine.framgia.vn/issues/4122
me fragment

### CODE REVIEW

**Comment by duccnv-1684** [comment]
ready

File: app/src/main/res/layout/fragment_me.xml [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+            android:src="@drawable/image_recent_default"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/text_full_name"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_me.xml]: 36 .. 39 42 hardcode

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by duccnv-1684** [comment]
ready

File: app/src/main/java/com/framgia/f_talk/util/di/builder/ActivityBuilder.java [hunk 2]
```diff
@@ -36,6 +37,6 @@
     abstract CreateAccountActivity bindCreateAccountActivity();
 
     @ContributesAndroidInjector(modules = {HomeActivityModule.class, RecentFragmentProvider.class,
-            GroupFragmentProvider.class})
+            GroupFragmentProvider.class,MeFragmentProvider.class})
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/f_talk/util/di/builder/ActivityBuilder.java]: format code

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by duccnv-1684** [comment]
ready

**Pull request merged**